ios/FluentUI/Core/Colors.swift

```diff
@@ -367,117 +368,127 @@ public final class Colors: NSObject {
 
     // MARK: Base semantic
 
-    @objc public static let background1 = UIColor(light: .white, dark: .black, darkElevated: gray900)
```

Comment:
I do like how much better named the new semantic colors are, but I feel for anyone using this color palette. This is a lot of pain to try and figure out what to replace their older usages with. I'm guessing the changes to the gray palette above mean that we can't just leave these around as deprecated? I know the office mobile app team is using them relatively extensively in their OFMColors.swift file.

Reply:
yeah the problem is it isn't one to one so it is hard to keep them around. I took looked at the OFMColors.swift and when we integrate this to office. I would definitely update them one by one which is the price to take anyway....

Reply:
Ideally we can help them make the update as we update to a new version of the framework. Luckily the new names are absurdly better for comprehension 👍